### DISCUSSION TITLE
Fully qualified domain names (FQDN) instead of IP addresses for the servers in HTTP requests - local redirect mode on Windows

### DISCUSSION THREAD

**Discussion by maasj** [mitmproxy/mitmproxy#6669] [Q&A] [answered]
Hello,

I'm trying to use the new local redirect mode on Windows and my goal is to log the full URL of all HTTP and HTTPS requests made by any application on the system.  I'm well on my way, except that it seems like mitmproxy and mitmdump both only show the IP address of the server and not the fully qualified domain name (FQDN), which I'd rather see.  Is there anything built-in that would show the hostnames instead of IP addresses, or would I need to use an add-on or do post-processing of the logs to do the reverse DNS lookups?

Thank you for this marvelous project!

Jason

**Comment by mhils** [comment, marked as answer]
Have you tried enabling the [`showhost`](https://docs.mitmproxy.org/stable/concepts-options/#showhost) option? :)

**Reply by maasj** [reply]
🤦‍♂️ I'm sorry, I don't know how I missed that.  Thank you for your gracious response and for your work on a very useful project!
